scripts/simulate.js

```diff
@@ -204,8 +204,8 @@ async function leverageCycle(amount, noGasTracking, cycle) {
 
   const startingEth = await signer.getBalance();
   const ptBalance = await purchasePTs(amount);
-  const fiatBalance = await collateralizeForFiat();
-  const { daiBalance, effectiveFiatPrice, reserves0, reserves1} = await curveSwapFiatForDai();
+  const fiatDebtInDai = await collateralizeForFiat();
+  const {daiBalance,effectiveFiatPrice,reservesDai,reservesFiat,fiatPoolShare} = await curveSwapFiatForDai();
```

Comment:
Can you put spaces back in? :) It looks like you rejected some of the conflicts.